### PR TITLE
Cult reminders

### PR DESCRIPTION
**Pull request by ShiftyRail**
[tweak]

Heavily inspired by the concept of Security records. I have no doubts cult players will be intelligent enough to make use of such a feature.

You can use again a "communicate" rune to write a cult reminder. This works similarly to cult message, except that it is also stored in your memory AND that it will be stored in the memory of all new converts.

You should use these to tell them where your base is, what your general plan of action is, etc, etc.

:cl:
- tweak: Using a communicate rune mid-cast will allow you to write a "Cult reminder", which will be engraved in the memory of all present cultists and all new converts.

### CODE REVIEW

**Comment by DrSnips** [comment]
MY BODY IS ON THE ASTEROI

**Comment by aceedex** [comment]
Reminder: pee pee poo poo :>)

Neat pr tho. Love it

**Comment by Allspice** [comment]
spiffy

**Comment by boy2mantwicethefam** [comment]
Today I'll remind them

**Comment by Hoshi-chan** [comment]
Nice.

**Comment by Irradiation** [comment]
One day, Woody got Wood...

**Comment by DeityLink** [comment]
That's an interesting implementation of the idea I had during cult 3 development, except it was about using the Arcane Tome's special page to write info that anyone else with a tome can read. I think my solution is more diegetic and elegant (and prevents the issue of accidentally double-clicking a comms rune) but I never took the time to implement it.

**Comment by ShiftyRail** [comment]
Not every cultist has the time or the presence of mind to open a book ; said book can be lost.
These reminders are eternal.

The solution is arguably a bit too handwavey, but it's just a way for the cult to keep records without too much hassle. Compare using computers and using secHuDs to write notes, which one is the most consistently used ? HuDs, because they're easier to interact with. (unless the prep is missing but that's another use case.)

File: code/datums/gamemode/factions/bloodcult/bloodcult_runespells.dm [hunk 1]
```diff
@@ -393,7 +393,7 @@
 /datum/rune_spell/communication
 	name = "Communication"
 	desc = "Speak so that every cultists may hear your voice."
-	desc_talisman = "Use it to write and send a message to all followers of Nar-Sie."
+	desc_talisman = "Use it to write and send a message to all followers of Nar-Sie. When in the middle of a ritual, use it to transmit a message that will be remembered at all."
```
Comment (Kurfursten) [inline code/datums/gamemode/factions/bloodcult/bloodcult_runespells.dm]: remembered by all?

**Comment by Hoshi-chan** [comment]
I like Deity's idea for implementation better, tbh.

**Comment by Kurfursten** [comment]
FIX IT

**Pull request merged**